lockfile/src/golang.rs

```diff
@@ -0,0 +1,54 @@
+use std::ffi::OsStr;
+use std::path::Path;
+
+use anyhow::{anyhow, Context};
+use nom::error::convert_error;
+use nom::Finish;
+use phylum_types::types::package::PackageType;
+
+use super::parsers::go_sum;
```

Comment:
```suggestion
use crate::parsers::go_sum;
```

I'd assume you've just copied this from another parser, but there's really no need for this confusing indirection when it's literally just in the crate root anyway.